release/CMakeLists.txt

```diff
@@ -15,10 +15,12 @@ endif()
 
 configure_file(tigervnc.iss.in tigervnc.iss)
 
-add_custom_target(installer
-  iscc -o. ${INST_DEFS} -F${CMAKE_PROJECT_NAME}${INST_SUFFIX}-${VERSION} tigervnc.iss
-  DEPENDS vncviewer
-  SOURCES tigervnc.iss)
+if(BUILD_VIEWER)
```

Comment:
Probably easier to make this entire directory conditional on the viewer being built.